service-tests/helpers/validators.js

```diff
@@ -15,6 +15,8 @@ const isStarRating = withRegex(/^[\u2605\u2606]{5}$/);
 
 const isMetric = withRegex(/^[0-9]+[kMGTPEZY]?$/);
 
+const isMetricOverPeriod = withRegex(/^[0-9]+[kMGTPEZY]?\/.+$/);
```

Comment:
Since this is for downloads, I'm guessing period means week, month, year. Could you make that explicit in the regex, and also call it `isMetricOverTimePeriod`?

Would you be up for updating the other service tests where this regex could be used?

Reply:
Sure, I can go through the other tests and open a new "clean up" pull request.